cip/CIP2016-07-07-Type-conversion-functions.adoc

```diff
@@ -0,0 +1,79 @@
+= CIP2016-07-07 - Type conversion functions
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Mats Rydberg <mats@neotechnology.com>
+
+[abstract]
+.Abstract
+--
+Cypher contains three type conversion functions `toString()`, `toInt()`, and `toFloat()`.
+This CIP suggests introducing a more comprehensive rule for specifying the existence of these functions, following a simple `toX()` pattern, where `X` is the name of the type.
+Further, this CIP suggests expanding this rule to encompass all scalar types.
+--
+
+toc::[]
+
+
+== Motivation
+
+The `toInt()` function indicates conversion to something called `Int`, which does not exist in Cypher.
```

Comment:
Does it really indicate conversion to a _type_ though? It does indicate conversion to `Int`, but doesn't say much about what `Int` may be. I changed it to:
```
The `toInt()` function indicates conversion to `Int`, which is neither a type nor any other known Cypher construct.
```
WDYT?